test/unit/testBase.js

```diff
@@ -61,19 +61,25 @@ before('Setting up example directory', function beforeHook(done) {
   delete package.dependencies;
   package.devDependencies['@adobe/parcel-plugin-htl'] = 'file:./../..';
 
+  fs.removeSync('./test/example/package-lock.json');
```

Comment:
I somehow don't like the way this is initialized above. I'd rather have a fixed `package.json` for the tests. so it 1. serves as example on what to provide; 2. isn't coupled to the implementation's package.json

Reply:
I had a separate `package.json` for the tests before, but then we had dependency versions going out of sync between the plugin and its tests.